pandas/_testing.py

```diff
@@ -1217,13 +1347,15 @@ def assert_frame_equal(
     check_index_type="equiv",
     check_column_type="equiv",
     check_frame_type=True,
-    check_less_precise=False,
+    check_less_precise=no_default,
```

Comment:
you can just use None here, no?

Reply:
Sure. I was just following your suggestion:

> you can just set the default for check_less_precise = lib.no_default (grep for it)

Which seemed a better fit when deprecating a parameter. Should I change it to `None` or keep with the `no_default`? :smile: